cmd/kubeadm/app/master/apiclient.go

```diff
@@ -103,6 +104,15 @@ func CreateClientAndWaitForAPI(adminConfig *clientcmdapi.Config) (*clientset.Cli
 	return client, nil
 }
 
+func isNodeReady(node *api.Node) bool {
```

Comment:
Just seems like `pkg/api/v1` doesn't have a `resource_helpers.go` file that `pkg/api` has.
We should consider adding `pkg/api/v1/resource_helpers.go`, yes